src/Logging.XUnit/XUnitLogger.cs

```diff
@@ -264,28 +267,53 @@ private static string GetLogLevelString(LogLevel logLevel)
         private static void GetScopeInformation(StringBuilder builder)
         {
             var current = XUnitLogScope.Current;
-            string scopeLog;
-            int length = builder.Length;
 
+            var stack = new Stack<XUnitLogScope>();
             while (current != null)
             {
-                if (length == builder.Length)
-                {
-                    scopeLog = $"=> {current}";
-                }
-                else
+                stack.Push(current);
+                current = current.Parent;
+            }
+
+            var depth = 0;
+            string DepthPadding(int depth) => new string(' ', depth * 2);
+
+            while (stack.Any())
+            {
+                var elem = stack.Pop();
+                foreach (var property in StringifyScope(elem))
                 {
-                    scopeLog = $"=> {current} ";
+                    builder.AppendLine($"{MessagePadding}{DepthPadding(depth)}=> {property}");
                 }
 
-                builder.Insert(length, scopeLog);
-                current = current.Parent;
+                depth++;
             }
+        }
 
-            if (builder.Length > length)
+        /// <summary>
+        /// Returns one or more stringified properties from the log scope.
+        /// </summary>
+        /// <param name="scope">The <see cref="XUnitLogScope"/> to stringify.</param>
+        /// <returns>An enumeration of scope properties from the current scope.</returns>
+        private static IEnumerable<string> StringifyScope(XUnitLogScope scope)
+        {
+            if (scope.State is IEnumerable<KeyValuePair<string, object>> pairs)
+            {
+                foreach (var pair in pairs)
+                {
+                    yield return $"{pair.Key}: {pair.Value}";
```

Comment:
As it's a simple concat I would have thought it _slightly_ more performant to not use interpolation:

```suggestion
                    yield return pair.Key + ": " + pair.Value;
```